src/index.html

```diff
@@ -7,8 +7,34 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="1" class="nav__link is-active">Apple</a></li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="1" class="nav__link is-active">
              Apple
            </a>  
```
<img width="546" alt="Screenshot 2023-01-25 at 18 14 02" src="https://user-images.githubusercontent.com/53228259/214616087-2737a63c-6e7c-46de-9393-2a02a53a1858.png">

Reply:
and change href to "/" in all a tags